HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -337,18 +302,24 @@ def self.apply_natural_ventilation_and_whole_house_fan(model, weather, vent_whf,
       vent_program.addLine("Set Tnvsp = #{UnitConversions.convert(73.0, 'F', 'C')}") # Assumption when no HVAC system
     end
     vent_program.addLine("Set NVavail = #{nv_avail_sensor.name}")
-    vent_program.addLine("Set WHFavail = #{whf_avail_sensor.name}")
     vent_program.addLine("Set ClgSsnAvail = #{nv_clg_ssn_sensor.name}")
     vent_program.addLine('If (Wout < MaxHR) && (Phiout < MaxRH) && (Tin > Tout) && (Tin > Tnvsp) && (ClgSsnAvail > 0)')
-    vent_program.addLine("  Set WHF_Flow = #{UnitConversions.convert(whf_cfm, 'cfm', 'm^3/s')}")
+    vent_program.addLine('  Set WHF_Flow = 0')
+    vent_fans_whf.each do |vent_whf|
+      vent_program.addLine("Set WHF_Flow = WHF_Flow + #{UnitConversions.convert(vent_whf.rated_flow_rate, 'cfm', 'm^3/s')} * #{whf_avail_sensors[vent_whf.id].name}")
```

Comment:
Even though it doesn't make it to the IDF, please try to indent the EMS lines to show how nested they are. I.e., `"Set WHF_Flow ...` -> `"  Set WHF_Flow`